math/minuit2/src/MnPosDef.cxx

```diff
@@ -43,7 +43,7 @@ MinimumError MnPosDef::operator()(const MinimumError &e, const MnMachinePrecisio
    }
    //   std::cout<<"MnPosDef init matrix= "<<err<<std::endl;
 
-   double epspdf = std::max(1.e-6, prec.Eps2());
+   double epspdf = std::max(1.e-12, prec.Eps2()); // should this lower bound be configurable?
```

Comment:
I agree this should be a configurable parameter. It is important to decide later in the code (line 85, when looking at the eigenvalues, if the matrix need to be made positive defined by adding some extra values to the diagonal.
It is clear that we higher statistics there will be smaller and smaller eigenvalue of the covariance matrices and the spread of the eigenvalues will increase. 
Note that the Fortran implementation of Minuit (and TMInuit) uses 1.E-6 as value for this parameter.